app/matplotlib.ipynb

```diff
@@ -194,21 +198,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
     "\n",
-    "# write your code here"
+    "fig, ax = plt.subplots()\n",
+    "ax.pie(market_share, labels=labels, autopct='%1.1f%%', startangle=90)\n",
+    "ax.set_title(\"Pie plot\")"
```

Comment:
The pie plot is missing the `plt.show()` function call to display the plot. Please add `plt.show()` at the end of this cell to ensure the plot is displayed.